Modules/Session/classes/class.ilObjSessionGUI.php

```diff
@@ -1879,8 +1879,8 @@ protected function load()
 			$this->object->getFirstAppointment()->setDaysOffset((int)$offset);
 			$this->object->getFirstAppointment()->setStart($start);
 			$this->object->getFirstAppointment()->setEnd($end);
-			$this->object->getFirstAppointment()->setStartingTime($start->getUnixTime()+$start->getUTCOffset());
```

Comment:
Notice, that the user-timezone is set at rendering, e.g. in form or other outputs. Reading and writing to db, however, has to take place consistently. in this case we (or rather the creator of this code) picked utc.